--- jobOptions_tagbb.py	2016-07-30 00:06:33.000000001 +0200
+++ jobOptions_holistic.py	2016-07-30 00:12:03.000000001 +0200
@@ -266,8 +266,12 @@
   alg.EssentialInfo=ONLYEssentialInfo
   alg.DoMSV     =DoMSV
   alg.Rel20     =True
+
   alg.SubjetInfo = True
+  alg.DumpTrackCovariance = True
+  alg.DumpCaloInfo = True
   #alg.CleanParentJets = True
+
   alg.JetCleaningTool.CutLevel= "LooseBad" 
   alg.JetCleaningTool.DoUgly  = True
   if not doComputeReference: algSeq += alg
